content/doc/book/blueocean/create-pipeline.adoc

```diff
@@ -0,0 +1,109 @@
+---
+layout: section
+title: Creating Pipelines in Blue Ocean
+---
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
+
+= Creating Pipelines in Blue Ocean
+
+Blue Ocean makes it easy to create new Pipelines or create Pipelines from an existing
```

Comment:
I generally agree with the adherence to the phrasing used in the UI, if you're not keen to use the term "import" then I think the paragraph needs some more fluffy prose to help distinguish the two possible states to the reader.

Perhaps something like "Blue Ocean makes it easy to create new Pipelines in Jenkins from an existing `Jenkinsfile`, or by creating a new `Jenkinsfile` with the Blue Ocean Pipeline Editor."?



Also, [people use import in a way which you might not appreciate](https://www.google.com/search?q=importing+from+sql+file&oq=importing+from+sql+file) :smiling_imp: